app/multitenant/aws_collector.go

```diff
@@ -277,7 +277,8 @@ func (c *awsCollector) getReports(reportKeys []string) ([]report.Report, error)
 		if store == nil {
 			continue
 		}
-		found, missing, err := store.FetchReports(missing)
+		found, newMissing, err := store.FetchReports(missing)
+		missing = newMissing
```

Comment:
I wonder if it'd be more idiomatic / clearer to initialize `found` and `err` outside of the loop and set them here with `=`. Genuine question.